app/Resources/views/blog/rss.xml.twig

```diff
@@ -0,0 +1,23 @@
+{% spaceless %}
+<?xml version="1.0" encoding="UTF-8" ?>
+<rss version="2.0">
+    <channel>
```

Comment:
What if we add these two tags to the `<channel>` tag?

```twig
<pubDate>{{ 'now'|date('r') }}</pubDate>
<lastBuildDate>{{ (posts|last).publishedAt|default('now')|date('r') }}</lastBuildDate>
```

* pubDate: The original publication date for the channel or item. (optional)
* lastBuildDate: The most recent time the content of the channel was modified. (optional)

They are totally optional, so if you don't agree that's OK!